db/schema.rb

```diff
@@ -37,6 +37,9 @@
     t.string   "digital_cico_number"
     t.datetime "created_at",          null: false
     t.datetime "updated_at",          null: false
+    t.string   "dc_num"
+    t.index ["dc_num"], name: "index_books_on_dc_num", unique: true
```

Comment:
Done; I noticed them but couldn't find an example of a migration that would remove them (or how they got there in the first place)